### PR TITLE
Prac 09 feedback

### PR DESCRIPTION
**Pull request by Jordan-Whiteside**
@tycabassi Could you please check my class and function design as well as my variable naming. Thanks!

### CODE REVIEW

File: prac_09/silver_service_taxi_test.py [hunk 1]
```diff
@@ -7,4 +7,4 @@
 assert my_taxi.drive(18) == 18
 print(my_taxi.get_fare())
 assert my_taxi.get_fare() == 48.8
```
Comment (tycabassi) [inline prac_09/silver_service_taxi_test.py:9]: Good use of testing here

File: prac_09/band.py [hunk 2]
```diff
@@ -5,16 +5,19 @@ class Band:
...
 
-
     def add(self, musician):
+        """Add musician to band."""
         self.musicians.append(musician)
 
     def play(self):
+        """Play instrument for each band member."""
         for musician in self.musicians:
             print(musician.play())
```
Comment (tycabassi) [inline prac_09/band.py:23]: Is printing musician.play() necessary?

File: prac_09/unreliable_car_test.py [hunk 3]
```diff
@@ -1,7 +1,7 @@
-"""unreliable car test."""
+"""Unreliable car test."""
 
 from prac_09.unreliable_car import UnreliableCar
 
 my_unreliable_car = UnreliableCar("Shit box", 1000, 50)
```
Comment (tycabassi) [inline prac_09/unreliable_car_test.py:5]: I found the name of the unreliable car quite humourous

File: prac_09/taxi_simulator.py [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+    display_taxis(taxis)
+
+
+def display_taxis(taxis):
+    """Display taxis."""
+    for i, taxi in enumerate(taxis):
+        print(f"{i}, {taxi}")
+
+
+def get_taxi(taxis, current_taxi):
```
Comment (tycabassi) [inline prac_09/taxi_simulator.py:41]: Does this just get a taxi or does it get a valid taxi?

**Review by tycabassi — COMMENTED** [review]
Hey Jordan,
You killed it this week all the code work and you used classes well and the the naming for the most part is good. Good work!
Kind regards, Ty Cabassi

File: prac_09/taxi_simulator.py (hunk 4 above)
Comment (Jordan-Whiteside) [reply prac_09/taxi_simulator.py:41]: I am getting a valid taxi here though I would consider that a side effect. You need to break out of the function if the user inputs something wrong. If you look at the solution it goes back to main on a wrong input. If I have a `get_valid_number` function then it loops until a valid number is entered but that's not what I  want. Usually I would use a `get_valid_number/taxi` function here but it seem unnecessary and went against the sample output.

File: prac_09/band.py (hunk 2 above)
Comment (Jordan-Whiteside) [reply prac_09/band.py:23]: Yeah good catch. I'll fix it up! Thank you!

**Review by Jordan-Whiteside — COMMENTED** [review]
Thanks I appreciate the review!

**Pull request merged**